packages/eas-cli/src/project/ios/__tests__/target-test.ts

```diff
@@ -21,13 +33,24 @@ describe(getApplePlatformFromSdkRoot, () => {
   });
   test('existing SDKs work with the function', () => {
     // sdks from `xcodebuild -showsdks`
-    expect(getApplePlatform('iphoneos14.4')).toBe(ApplePlatform.IOS);
-    expect(getApplePlatform('iphonesimulator14.4')).toBe(ApplePlatform.IOS);
-    expect(getApplePlatform('driverkit.macosx20.2 ')).toBe(ApplePlatform.MAC_OS);
-    expect(getApplePlatform('macosx11.1')).toBe(ApplePlatform.MAC_OS);
-    expect(getApplePlatform('appletvos14.3')).toBe(ApplePlatform.TV_OS);
-    expect(getApplePlatform('appletvsimulator14.3')).toBe(ApplePlatform.IOS);
-    expect(getApplePlatform('watchos7.2')).toBe(ApplePlatform.IOS);
-    expect(getApplePlatform('watchsimulator7.2')).toBe(ApplePlatform.IOS);
+    expect(getApplePlatformWithSdkRoot('iphoneos14.4')).toBe(ApplePlatform.IOS);
+    expect(getApplePlatformWithSdkRoot('iphonesimulator14.4')).toBe(null);
+    expect(getApplePlatformWithSdkRoot('driverkit.macosx20.2 ')).toBe(ApplePlatform.MAC_OS);
+    expect(getApplePlatformWithSdkRoot('macosx11.1')).toBe(ApplePlatform.MAC_OS);
+    expect(getApplePlatformWithSdkRoot('appletvos14.3')).toBe(ApplePlatform.TV_OS);
+    expect(getApplePlatformWithSdkRoot('appletvsimulator14.3')).toBe(null);
+    expect(getApplePlatformWithSdkRoot('watchos7.2')).toBe(null);
+    expect(getApplePlatformWithSdkRoot('watchsimulator7.2')).toBe(null);
+  });
+});
+
+describe(getApplePlatformFromDeviceFamily, () => {
+  test('apple platform can be obtained from device family types', () => {
```

Comment:
Most common case will be "1,2" (universal iPhone and iPad). Can you add a test for that?